airbyte/metadata.csv

```diff
@@ -0,0 +1,52 @@
+airbyte.worker.attempt.created,count,,,,Increments when a new attempt is created. One is emitted per attempt.,0,airbyte,attempt created,
+airbyte.worker.attempt.created_by_release_stage,count,,,,Increments when a new attempt is created. Attempts are double counted as this is tagged by release stage.,0,airbyte,attempt created by release stage,
+airbyte.worker.attempt.failed_by_failure_origin,count,,,,"Increments for every failure origin a failed attempt has. Since a failure can have multiple origins, a single failure can be counted more than once. Tagged by failure origin and failure type.",0,airbyte,attempt failed by failure origin,
+airbyte.worker.attempt.failed_by_release_stage,count,,,,Increments when an attempt fails. Attempts are double counted as this is tagged by release stage.,0,airbyte,attempt failed by release stage,
+airbyte.worker.attempt.succeeded_by_release_stage,count,,,,Increments when an attempts succeeds. Attempts are double counted as this is tagged by release stage.,0,airbyte,attempt succeeded by release stage,
+airbyte.worker.destination_buffer_size,gauge,,,,The size of the replication worker destination buffer queue.,0,airbyte,worker destination buffer size,
+airbyte.worker.destination_message_read,count,,,,Whenever a message is read from the destination.,0,airbyte,worker destination message read,
+airbyte.worker.destination_message_sent,count,,,,Whenever a message is sent to the destination.,0,airbyte,worker destination message sent,
+airbyte.worker.job.cancelled_by_release_stage,count,,,,Increments when a job is cancelled. Jobs are double counted as this is tagged by release stage.,0,airbyte,job cancelled by release stage,
+airbyte.worker.job.created_by_release_stage,count,,,,Increments when a new job is created. Jobs are double counted as this is tagged by release stage.,0,airbyte,job created by release stage,
+airbyte.worker.job.failed_by_release_stage,count,,,,Increments when a job fails. Jobs are double counted as this is tagged by release stage.,0,airbyte,job failed by release stage,
+airbyte.worker.job.succeeded_by_release_stage,count,,,,Increments when a job succeeds. Jobs are double counted as this is tagged by release stage.,0,airbyte,job succeeded by release stage,
+airbyte.worker.kube_pod_process_create_time_millisecs,distribution,,millisecond,,Time taken to create a new kube pod process.,0,airbyte,kube pod process create time millisecs,
+airbyte.worker.notifications_sent,count,,,,Number of notifications sent.,0,airbyte,notifications sent,
+airbyte.worker.replication_bytes_synced,count,,byte,,Number of bytes synced during replication.,0,airbyte,replication bytes synced,
+airbyte.worker.replication_records_synced,count,,record,,Number of records synced during replication.,0,airbyte,replication records synced,
+airbyte.worker.source_buffer_size,gauge,,,,The size of the replication worker source buffer queue.,0,airbyte,worker source buffer size,
```

Comment:
is this in byte?

Reply:
`num_records`, so I added the `record` number